deps/rabbit/src/rabbit_khepri.erl

```diff
@@ -295,6 +295,21 @@ wait_for_leader(Timeout, Retries) ->
             throw(Reason)
     end.
 
+wait_for_register_projections() ->
+    wait_for_register_projections(retry_timeout(), retry_limit()).
+
+wait_for_register_projections(_Timeout, 0) ->
+    exit(timeout_waiting_for_khepri_projections);
+wait_for_register_projections(Timeout, Retries) ->
+    rabbit_log:info("Waiting for Khepri projections for ~tp ms, ~tp retries left",
+                    [Timeout, Retries - 1]),
+    try
+        register_projections()
```

Comment:
This would retry registering all projections again if it fails. I think that's probably ok since I expect the first projection would be the only one to time out but we could make this a little more clever and fold over the list of projections to register and the retry number. Feel free to ignore though, it seems very unlikely to be hit in practice